reference/glossary.md

```diff
@@ -105,6 +109,10 @@ Proxy Virtual Machine.
 A type of [VM](#vm) that proxies network access for other VMs. 
 Typically, a ProxyVM sits between a NetVM and another VM (such as an AppVM or a TemplateVM) that requires network access.
 
+TempalateBasedProxyVM
+---------------------
+Any [ProxyVM](#proxyvm) that depends on a TemplateVM for its root filesystem.
```

Comment:
Same. Even in R3.x all ProxyVMs were based on TemplateVMs, and there are no ProxyVMs in R4.0.